File: Python Basics/Homeworks and Labs Exercises/02Conditional Statements/Conditional Statements Exercise/Godzilla_vs_Kong.py
budget = float(input())     # Budget
number_statistics = int(input())    # Number statistics:
price_statistics_dress = float(input())     # Cena obleklo edin statist: "

price_dress_total = price_statistics_dress * number_statistics
dekor = budget * 0.10
expenses = price_dress_total + dekor

if number_statistics > 150:
    price_dress_total -= price_dress_total * 0.10

if expenses > budget:
    needed_money = expenses - budget
    print("Not enough money!")
    print(f"Wingard needs {needed_money:.2f} leva more.")
elif expenses <= budget:
    left_money = budget - expenses
    print("Action!")
    print(f"Wingard starts filming with {left_money:.2f} leva left.")

File: Java Basics/Homeworks and Labs Exercises/02Conditional Statements/Conditional Statements Exercises/GodzillaVSKong.java
package JAVA.PROGRAMMING_BASICS.Conditional_Statements.Excercises;

import java.util.Scanner;

public class GodzillaVSKong {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        //  Декорът за филма е на стойност 10% от бюджета.
        // При повече от 150 статиста, има отстъпка за облеклото на стойност 10%.

        double budget = Double.parseDouble(scanner.nextLine());
        double decor = budget * 0.1;

        int numberStatists = Integer.parseInt(scanner.nextLine());
        double priceDressStatists = Double.parseDouble(scanner.nextLine());
        double totalDress = numberStatists * priceDressStatists;

        if (numberStatists > 150) {
            totalDress = totalDress - ( totalDress * 0.1);
        }

        double totalPriceFilm = decor + totalDress;

        if (budget >= totalPriceFilm  ) {
            System.out.println("Action!");
            System.out.printf("Wingard starts filming with %.2f leva left.", budget - totalPriceFilm);
        }
            else {
            System.out.println("Not enough money!");
            System.out.printf("Wingard needs %.2f leva more.", totalPriceFilm - budget);
        }

    }
}
